pkg/linters/conversions/rules.go

```diff
@@ -0,0 +1,253 @@
+package conversions
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"regexp"
+	"slices"
+	"strconv"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/deckhouse/dmt/pkg/errors"
+)
+
+const (
+	conversionsFolder = "openapi/conversions"
+)
+
+var regexVersionFile = regexp.MustCompile(`^v([1-9]\d{0,2})\.ya?ml$`)
+
+type conversion struct {
+	Version     *int         `yaml:"version,omitempty"`
+	Description *description `yaml:"description,omitempty"`
+}
+
+type description struct {
+	English string `yaml:"en,omitempty"`
+	Russian string `yaml:"ru,omitempty"`
+}
+
+func checkModuleYaml(moduleName, modulePath string) errors.LintRuleErrorsList {
+	result := errors.LintRuleErrorsList{}
+
+	if slices.Contains(Cfg.SkipCheckModule, moduleName) {
+		return result
+	}
+
+	folder := filepath.Join(modulePath, conversionsFolder)
+
+	stat, err := os.Stat(folder)
+	if err != nil && !os.IsNotExist(err) {
+		return result
```

Comment:
If we have an error other than "non-existent", shouldn't we show it in the linter output? To understand that the processing was not carried out to the end.